build.gradle

```
@@ -29,12 +29,21 @@ allprojects {
       if (unityClassesJar == null || unityClassesJar.isEmpty()) {
           unityClassesJar = System.getenv("UNITY_CLASSES_JAR")
       }
-      if (unityClassesJar == null || unityClassesJar.isEmpty()) {
-          unityClassesJar ='/Applications/Unity/PlaybackEngines/AndroidPlayer/Variations/mono/Release/Classes/classes.jar'
+
+      if (unityClassesJar == null || unityClassesJar.isEmpty() || !file(unityClassesJar).exists()) {
+            new ByteArrayOutputStream().withStream { os ->
+                def result = exec {
+                    executable = 'bash'
+                    args = ['-c', "find /Applications/Unity/Hub/Editor/" + airshipProperties.unityVersion + "/PlaybackEngines/AndroidPlayer -name classes.jar -type f | tail -1"]
```

You need the "mono-elease" one, right?
```suggestion
                    args = ['-c', "find /Applications/Unity/Hub/Editor/" + airshipProperties.unityVersion + "/PlaybackEngines/AndroidPlayer/Variations/mono/Release/Classes -name classes.jar -type f"]
```
Given that, actually, I think you don't even need to do the search. Just use "/Applications/Unity/Hub/Editor/" + airshipProperties.unityVersion + "/PlaybackEngines/AndroidPlayer/Variations/mono/Release/Classes/classes.jar"

---

And UnityExe can be simplified in the same way.